docs/ingestion/native-batch-input-source.md

```diff
@@ -773,7 +773,7 @@ The following is an example of a Combining input source spec:
         "delegates" : [
          {
           "type": "local",
-          "filter" : "*.csv",
+          "objectGlob" : "**.csv",
```

Comment:
does local input source has `objectGlob`?

Reply:
Ah, I must have searched and replaced too agressively.